docs/API/plain_rules.md

```diff
@@ -177,10 +178,23 @@ fields:
 Is recommended to set checkInterval at least double of reportInterval. Howeer, note that a very demanding value of
 checkInterval could impact on performance.
 
-<a name="actions"></a>
+## Non signal actions
+
+The following virtual attributes are available to be used in non signal template actions:
+-   **service**: service of rule
+-   **subservice**: subservice of rule
+-   **ruleName**: name of the rule
+-   **reportInterval**: time to see an entity as silent
```

Comment:
Maybe it's a good idea to include the detail:

```suggestion
-   **reportInterval**: time to see an entity as silent (in other words, how much time has to pass since last entity update to consider that entity is in "no signal" situation so the rule is triggered)
```

Also in the reportInterval definition in the "No signal conditions" section.